swift/Sources/Addresses/CoinType+Address.swift

```diff
@@ -96,7 +96,7 @@ public extension CoinType {
             return Set([P2SHPrefix.dash.rawValue, P2PKHPrefix.dash.rawValue])
         case .zcoin:
             return Set([P2SHPrefix.zcoin.rawValue, P2PKHPrefix.zcoin.rawValue])
-        case .zcash:
+        case .zelcash, .zcash:
             return Set([P2SHPrefix.zcashT.rawValue, P2PKHPrefix.zcashT.rawValue])
```

Comment:
what's the decision behind making the same prefix as zcash?

Reply:
Hello vikmeup.
We had no reason to change it as the address prefix makes sense. t for transparent, z for zk-snarks private. Our Acadia "Sapling" private addresses differ from Zcash. But transparent are the same.
Best regards,
Valter